components/screens/full-pages/ImportRecoveryPhrase.tsx

```diff
@@ -79,7 +79,7 @@ export default function ImportRecoveryPhrase() {
   };
 
   return (
-    <div className="flex h-[52rem] w-[41rem] flex-col items-stretch gap-4 rounded-3xl bg-icy-blue-950">
+    <div className="flex max-h-[56rem] w-[41rem] flex-col items-stretch gap-4 rounded-3xl bg-icy-blue-950 px-4">
```

Comment:
It fixes the issue:
![image](https://github.com/user-attachments/assets/db12083d-abed-407c-8704-3b14a62b4ccb)